backend/apps/users/models.py

```diff
@@ -53,6 +45,18 @@ def generate_email_verification_code(self):
             {'verification_code': verification.code}
         )
 
+class Doctor(User):
```

Comment:
This model is not related with USER and scr_access and online_consultation_capability fields are not needed.